packages/www/pages/app/admin/user/[id].tsx

```diff
@@ -194,7 +195,10 @@ const ID = () => {
                   </Box>
                   <Box sx={{ m: "0.4em" }}>Viewer limit</Box>
                   <Box sx={{ m: "0.4em" }}>
-                    {userInfo.viewerLimit ? userInfo.viewerLimit : 50000}
+                    {userInfo.viewerLimit ??
+                      (userInfo?.stripeProductId === "prod_O9XuIjn7EqYRVW"
+                        ? 30
```

Comment:
should this be 30000?

Reply:
This is just the value displayed on the admin page for `Sandbox` users, which is 30!
The value are theres:
- Sandbox `30`
- Enterprise `20_000`
- All the others `10_000`

Reply:
Ah right, of course!